Tests/ML/utils/test_augmentation.py

```diff
@@ -25,6 +25,9 @@
 valid_labels = np.zeros((number_of_classes,) + image_size)
 for c in range(number_of_classes):
     valid_labels[c, class_assignments == c] = 1
+# Since we have 5 classes and all ground truth class labels are provided, initialize 'missing_labels_list' to
+# to 'False' with  length 5
```

Comment:
See my other comment - please do not add comments that just repeat what the code does. If there's a special twist to why you are choosing a particular value, sure, but here I feel that's wasted space.